integration/flink/src/main/java/io/openlineage/flink/utils/KafkaUtils.java

```diff
@@ -0,0 +1,26 @@
+/*
+/* Copyright 2018-2023 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+package io.openlineage.flink.utils;
+
+import io.openlineage.client.utils.DatasetIdentifier;
+import java.util.Properties;
+
+public class KafkaUtils {
+
+  private static final String KAFKA_DATASET_PREFIX = "kafka://";
+
+  /**
+   * Generate dataset identifier based on bootstrap servers in properties and topic name
+   *
+   * @param properties
+   * @param topic
+   * @return
+   */
+  public static DatasetIdentifier datasetIdentifierOf(Properties properties, String topic) {
+    String bootstrapServers = properties.getProperty("bootstrap.servers");
```

Comment:
thinking if we should normalize the bootstrap servers property?
for example, strip out `https://` prefixes and trailing `/`

Reply:
I thought bootstrap servers is just a comma separated list of hosts and cannot contain `http://`.

The bigger problem is that the list of bootstrap servers can change over time, while we still would like to have the same dataset, like a topic on a Kafka cluster. We'll need to revisit this some day.

Reply:
oh, I think you're right about that

Reply:
and yes, broker servers might not be the best identifier for a kafka namespace. but to your point, out of scope for this PR.